plugins/cost-insights/src/components/CostOverviewCard/CostOverviewCard.tsx

```diff
@@ -15,42 +15,41 @@
  */
 
 import React from 'react';
-import { Box, Card, CardContent, Divider } from '@material-ui/core';
-import CostOverviewChart from '../CostOverviewChart';
-import CostOverviewChartLegend from '../CostOverviewChartLegend';
+import { Box, Card, CardContent, Divider, useTheme } from '@material-ui/core';
+import CostGrowth from '../CostGrowth';
+import CostOverviewChart from './CostOverviewChart';
 import CostOverviewHeader from './CostOverviewHeader';
+import LegendItem from '../LegendItem';
 import MetricSelect from '../MetricSelect';
 import PeriodSelect from '../PeriodSelect';
 import { useScroll, useFilters, useConfig } from '../../hooks';
 import { mapFiltersToProps } from './selector';
 import { DefaultNavigation } from '../../utils/navigation';
+import { formatPercent } from '../../utils/formatters';
 import {
-  ChangeStatistic,
-  DateAggregation,
-  Project,
-  Trendline,
+  Cost,
+  CostInsightsTheme,
+  MetricData,
   findAlways,
+  getComparedChange,
 } from '../../types';
 
-type CostOverviewCardProps = {
-  change: ChangeStatistic;
-  aggregation: Array<DateAggregation>;
-  trendline: Trendline;
-  projects: Array<Project>;
+export type CostOverviewCardProps = {
+  data: [Cost, MetricData | null];
 };
 
-const CostOverviewCard = ({
-  change,
-  aggregation,
-  trendline,
-}: CostOverviewCardProps) => {
-  const { metrics } = useConfig();
+const CostOverviewCard = ({ data }: CostOverviewCardProps) => {
+  const theme = useTheme<CostInsightsTheme>();
+  const config = useConfig();
   const { ScrollAnchor } = useScroll(DefaultNavigation.CostOverviewCard);
-  const { setDuration, setProject, metric, setMetric, ...filters } = useFilters(
+  const { setDuration, setProject, setMetric, ...filters } = useFilters(
     mapFiltersToProps,
   );
 
-  const { name } = findAlways(metrics, m => m.kind === metric);
+  // There should always be a daily cost metric but a comparison metric is optional.
+  const dailyCost = findAlways(config.metrics, m => m.kind === null);
+  const metric = config.metrics.find(m => m.kind === filters.metric);
+  const comparedChange = data[1] ? getComparedChange(data[0], data[1]) : null;
```

Comment:
Could do something like `const [costData, metricData] = data` to improve readability here

Reply:
It seems silly to combine them in a tuple prop if we're just going to destructure, can we just separate into two props?

Reply:
Gravitated toward a tuple because the component accepts "data" of the same type and cost and metric data are strongly mapped to each other in the cost chart. Plus, I'm generally trying to avoid having to define more props than necessary.

That said, I'd probably feel more strongly about keeping it as tuple if metricData was guaranteed, which it isn't. So I think the case for them as props is stronger.